package.json

```diff
@@ -7,14 +7,10 @@
     "bugs": "https://github.com/privacy-scaling-explorations/zk-kit.solidity/issues",
     "private": true,
     "scripts": {
-        "compile": "yarn workspaces foreach -Ap run compile",
-        "test": "yarn workspaces foreach -Ap run test:coverage",
-        "version:bump": "yarn workspace @zk-kit/${0}.sol version ${1} && yarn remove:stable-version-field ${0} && NO_HOOK=1 git commit -am \"chore(${0}): v${1}\" && git tag ${0}.sol-v${1}",
-        "version:publish": "yarn workspaces foreach -A --no-private npm publish --tolerate-republish --access public",
-        "version:release": "changelogithub",
-        "format": "prettier -c .",
-        "format:write": "prettier -w .",
-        "remove:stable-version-field": "ts-node scripts/remove-stable-version-field.ts ${0} && yarn format:write",
+        "test": "forge test",
```

Comment:
i decided to remove `remove:stable-version-field` and `version:publish` because this is handled by the ci, I think it is better not to  easily "expose"  such scripts in package.json